### PR TITLE
plugin manager uninstall plugin regardless of working directory

### LINKED ISSUE #10781: Can not remove my installed plugin
hi 
i installed my plugin . but when i want to remove my plugin or any other plugins , logstash said :
```
#my commands : /usr/share/logstash/bin/logstash-plugin remove logstash-filter-myplugin
#another command: /usr/share/logstash/bin/logstash-plugin remove logstash-filter-kv

ERROR: Operation aborted, cannot remove plugin, message: The path `/root/logstash-core` does not exist.
```
after i copy `/usr/share/logstash/logstash-core` to /root/logstash-core , error changed to :
```
ERROR: Operation aborted, cannot remove plugin, message: The path `/root/logstash-core-plugin-api` does not exist.
```

i think logstash dont know where is installed .

- Version: elk 7
- Operating System: ubuntu 18.4

### PR DESCRIPTION
**Pull request by kaisecheng**
## Release notes

Fix: allow plugin manager to remove plugin regardless of the current working directory

## What does this PR do?

change the removal procedure from relative path to absolute path

## Why is it important/What is the impact to the user?

This PR fixes an issue that was preventing user uninstalls plugin from the plugin manager when user is not in $LOGSTASH_HOME directory

## Author's Checklist

- [x] tested in macOS

## How to test this PR locally

Assuming your Logstash is not installed in home directory
In mac
run uninstall command outside $LOGSTASH_HOME
> cd ~
> $LOGSTASH_HOME/bin/logstash-plugin remove logstash-filter-kv

terminal should show "Successfully removed logstash-filter-kv"

## Related issues

Closes #10781

### CODE REVIEW

**Review by robbavey — CHANGES_REQUESTED** [review]
It would be nice to have tests for this - we have existing tests (https://github.com/elastic/logstash/blob/master/qa/acceptance/spec/shared_examples/cli/logstash-plugin/remove.rb#L40 and https://github.com/elastic/logstash/blob/master/qa/acceptance/spec/shared_examples/cli/logstash-plugin/uninstall.rb#L40), but they are currently skipped. 
I did a quick commit on a test branch, reinstating the test (ignoring the text part for now), and it looks like the underlying issue they were skipped for is fixed - and your change does seem to fix the tests as advertised.

**Review by roaksoax — APPROVED** [review]
LGTM!

**Comment by robbavey** [comment]
@kaisecheng The tests aren't run as part of standard CI - before committing can you push a new branch to the `elastic/logstash` repo and trigger a test manually from https://logstash-ci.elastic.co/job/elastic+logstash+master+multijob-acceptance/ using `build with parameters`, specifying your branch name in the parameters (you will need to login to jenkins to be able to do this).

File: qa/acceptance/spec/shared_examples/cli/logstash-plugin/remove.rb [hunk 1]
```diff
@@ -36,14 +36,13 @@
...
-    # Disabled because of this bug https://github.com/elastic/logstash/issues/5286
-    xcontext "when the plugin is installed" do
+    context "when the plugin is installed" do
       it "successfully removes it" do
         result = logstash.run_command_in_path("bin/logstash-plugin install logstash-filter-qatest")
         expect(logstash).to have_installed?("logstash-filter-qatest")
 
         result = logstash.run_command_in_path("bin/logstash-plugin remove logstash-filter-qatest")
-        expect(result.stdout).to match(/^Removing logstash-filter-qatest/)
+        # expect(result.stdout).to match(/^Removing logstash-filter-qatest/)
```
Comment (robbavey) [inline qa/acceptance/spec/shared_examples/cli/logstash-plugin/remove.rb:45]: Can we either remove the commented out line, or make this work? I'm fine with either - the text test is somewhat fragile, and the 'not_to_have_installed?` test is sufficient

**Review by robbavey — APPROVED** [review]
LGTM. Couple of nits on commented out code

**Review by robbavey — APPROVED** [review]
LGTM

**Pull request merged**